server/sonar-server/src/main/java/org/sonar/server/user/ws/UpdateAction.java

```diff
@@ -98,7 +97,7 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkLoggedIn().checkPermission(GlobalPermissions.SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not 'Administer System' permission... or is that the name we give to root?